branding/test.mk

```diff
@@ -12,3 +12,13 @@ export PORT=7777
 
 export MSI_PRODUCTCODE=e76baa9f-2bb2-49e5-b518-8a5b7d1cd084
 export OSX_IDPREFIX=org.jenkins-ci.test
+export AUTHOR=Bogus user <bogus@example.org>
+export LICENSE=MIT/X License, GPL/CDDL, ASL2
+export HOMEPAGE=http://test.jenkins-ci.org/
+export CHANGELOG_PAGE=http://test.jenkins-ci.org/changelog
+
+# figure out the directory of this file
+BRANDING_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
```

Comment:
this was coppied from 55f6a1d47bbd6891c9bb1e0ada78f429e6a36fd0#diff-d0946f7d822c0e5a2aff83a519016717R18  but it is utter garbage.

when adding an extra export to debug this 
`export MY_FILE_LIST=$(MAKEFILE_LIST)`
then `MY_FILE_LIST` contains  `Makefile branding/test.mk env/test.mk credentials/test.mk setup.mk`
the variable is set in `branding/test.mk` so that should be last - but it's not.

Make is evil.  Make must die.